www/app/App.js

```diff
@@ -36,22 +34,8 @@ export const App = ({ Component, pageProps }) => {
                 <meta name="msapplication-TileColor" content="#ffffff" />
                 <meta name="msapplication-config" content="/favicons/browserconfig.xml?v=M4KN2GElyG" />
                 <meta name="theme-color" content="#ffffff" />
```

Comment:
I know the above is specific to favicons and it might not directly be "seo", but maybe we could still use the Seo component to deal with this? 🤷

Reply:
That is a good point but I think we should keep it there and let me explain why.

We have instructions on [how to setup the favicon](https://next-with.moxy.tech/docs/welcome/instructions/#3-setup-favicon--manifest) and we recommend the use of [RealFaviconGenerator](https://realfavicongenerator.net/). This generator returns a similar HTML to what we have right now, so people just have to replace it. By moving that out it would be necessary to change it in two files, which is not hard but it's an extra step. Thoughts?

Reply:
Yes, the change would require that. Is not too much of an hassle since the usage of the Seo component is pretty straight forward and we would have a single place to manage these. Would like some other opinions on this.

Reply:
I agree with @paulobmarcos

Reply:
I think we can do this change then :+1:

Reply:
I changed it and updated the instructions. The meta tags related to the favicon are now being handled by the `Seo` component.
Let me know your thoughts.

Reply:
After speaking with Andre, we decided to let the meta and links tags related to the icons as it was.

Sorry for the extra work :+1: Can we roll back?

Reply:
so, what do you think to do a component like `<Seo>` for favicons?

```
<Head>
    <Favicons data={ faviconsData } />
    <Seo data={ seoData } />
</Head>
```